controllers/serviceController_int_test.go

```diff
@@ -35,7 +40,9 @@ var serviceWebApp = &corev1.Service{}
 var serviceAdditionalBytes []byte
 var serviceAdditional = &corev1.Service{}
 
-func init() {
+var serviceDeployment = &v1.Deployment{}
+
+func resetData() {
```

Comment:
The function name and the belonging package variable are cues that tests _may be_ interdependent on shared data. A better solution would include data and parse them per tests. This way no data are shared and no dependency is built.

Reply:
The resetData should be called for every test.